verticapy/connection/connect.py

```diff
@@ -125,6 +129,52 @@ def connect(section: str, dsn: Optional[str] = None) -> None:
         prev_conn.close()
     try:
         gb_conn.set_connection(vertica_connection(section, dsn), section, dsn)
+        print("Connected Successfuly!")
+    except OAuthTokenRefreshError as e:
+        error_message = str(e)
+
+        from verticapy.connection.write import new_connection
```

Comment:
why the import is inside the function?

Reply:
Otherwise there will be a circular import issue. 
in connect we are importing new_connection, and in new_connection we are importing connect.

Reply:
I am working on finding a solution to this.